tests/core/plan-env-file/test.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+. /usr/share/beakerlib/beakerlib.sh || exit 1
+
+rlJournalStart
+    rlPhaseStartSetup
+        rlRun 'pushd data'
+        rlRun 'set -o pipefail'
+    rlPhaseEnd
+
+    rlPhaseStartTest 'Check environment variable is available'
+        rlRun -s 'tmt run -vvv plan --name /plan/single'
```

Comment:
```suggestion
        rlRun -s 'tmt run -rvvv plan --name /plan/single'
```

Let's clean up after ourselves so that many run dirs are not left behind when running tests locally.

Reply:
Fixed in 9ecf336.